docs/usage/inference.rst

```diff
@@ -0,0 +1,168 @@
+Inference
+=========
+
+The inference functionality in mlGeNN allows you to run trained networks efficiently.
+This section covers how to use the inference compiler and its compiled networks.
+
+Creating an Inference Compiler
+----------------------------
+The ``InferenceCompiler`` provides the core functionality for running inference on
```

Comment:
Can you use e.g. ``:class:`.compilers.InferenceCompilers` `` so this links to the documentation?